src/store/selectors/index.js

```diff
@@ -0,0 +1,42 @@
+'use babel'
+
+import { createSelector } from 'reselect'
+import ruleDocURIs from 'eslint-rule-documentation'
+import r from '../../f-utils/mini-ramda'
```

Comment:
🔥 Only used in `selectDeepProp`, which isn't used.

Reply:
hmm, maybe some cleanup needed